src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_LENGTH = 1 << 4;
```

Comment:
```suggestion
    private static final int INITIAL_LENGTH = 16;
```

Reply:
we imitate the hashmap, hashmap uses this offset to give the INITIAL_LENGTH = 1 << 4

Reply:
@Hlib13 what the result of `1 << 4`?